lib/helpers.js

```diff
@@ -84,3 +87,47 @@ function messageToString(message) {
 
 	return message.toString();
 }
+
+/**
+ * Formats a number to the following specification:
+ * - Whole numbers have no decimal point
+ * - No trailing zeros
+ * - Max of two fraction digits
+ * - American English grouping and decimal separators are used (comma for thousands and period for
+ *   the decimal)
+ *
+ * Examples:
+ * 100 -> 100
+ * 2.789 -> 2.79
+ * 2000 -> 2,000
+ * 4582000.55000 -> 4,582,000.55
+ * 25.000000001 -> 25
+ */
+const format = new Intl.NumberFormat( 'en-US', { minimumFractionDigits: 0, maximumFractionDigits: 2 });
+function formatNumber(numStr) {
+	return format.format(numStr);
+}
+
+
+/**
+ * Logs a message to the console log with a timestamp prepended to it
+ *
+ * @param {*} message A message of any type
+ */
+function log(message) {
+	console.log(new Date().toLocaleString() + ' ' + messageToString(message));
+}
```

Comment:
I had a case where I noticed a problem days after it happened and I wasn't able to correlate the log messages to the original request. Adding the timestamp let me do that.

Good idea about prefixing the `id` variable. I'll go ahead and do that.

Reply:
BTW, I moved the `log` function to helpers because I was using it in another branch that I'm working on. What's the reason for only logging in bin/bot.js?

Reply:
Never mind. I'll live w/o the log function.

Reply:
> What's the reason for only logging in bin/bot.js?

So logging is centralized and commands are more sandboxed.